about.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+        <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+                    
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="style_about.css">
+        <title>The Concert's Webpage</title>
+    </head>
+<body>
+<!-- about -->
+<section id="about" class="about">
+    <!-- header -->
+    <header>
+        <nav class="navbar">
+            <a href="index.html"><img class="goat_logo_top" src="images/goat_top.PNG" alt="goat logo"><img></a>
+            <div>
+            <button class="hamburger">
+                <div class="bar"></div>
+            </button>
+        </div>
+            <ul class="menu" >
+                <li><a href="about.html">About</a></li>
```

Comment:
I agree, it can be confusing for the user. I added the Home button next to the About one. 
Nevertheless, this was a [specific requirement of the task](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md). That´s why its done that way!

![image](https://user-images.githubusercontent.com/108192399/217884184-450c38ed-107b-4c8a-b9c7-6c57e5726b76.png)

Reply:
- [ ] But it could be better if users can navigate between the two pages easily 👍